.config

```diff
@@ -1,2 +1,3 @@
 repo-owner = sanctuary-js
 repo-name = sanctuary-type-classes
+source-files = index.js bench/*.js
```

Comment:
I'm nervous about this change. It would mean that files in `bench/*.js` would be considered when generating the readme from code comments. It would be safer to provide `scripts/lint` and have it run `eslint` a second time.

Reply:
Ah yes. I was also nervous and was going to ask you about it.